htdocs/categories/index.php

```diff
@@ -177,6 +177,22 @@
 // Define data (format for treeview)
 $data = array();
 $data[] = array('rowid'=>0, 'fk_menu'=>-1, 'title'=>"racine", 'mainmenu'=>'', 'leftmenu'=>'', 'fk_mainmenu'=>'', 'fk_leftmenu'=>'');
+
+if (!empty($conf->global->CATEGORY_SHOW_COUNTS)) {
+	$entry = '<table class="nobordernopadding centpercent">';
+	$entry .=  '<tr class="" >';
+	$entry .=  '<td class=*left* >'.$langs->trans("Label").'</td>';
+	$entry .=  '<td class="left" width="40px">'.$langs->trans("Active").'</td>';
```

Comment:
What mean a nb of "active" ?

Reply:
Depending of élément, a statut or a status can have different values, how far i know dolibarr, only the object with a status more than 0 are open (active), instead of draft, resiliated or closed

for me it makes no sense to count objects that have been closed for years or are still at the draft stage. Even if the tag is still on the object.

As exemple, we use tag to know in witch section is one member. Some of them have resiliated for 10 years. When we display the count of member per section, it make no sens to count these member.

But, as the function should be able to read in all object on witch we can use a tag, I was not sur if there was a way to return the status type for each différent situation and translation.